dcm4che-core/src/main/java/org/dcm4che3/data/Attributes.java

```diff
@@ -1701,6 +1836,31 @@ public TimeZone getTimeZone() {
         return tz;
      }
 
+    public ZoneId getZoneId() {
+        // TODO we might want to store the ZoneId, instead of converting every time here
```

Comment:
The ZoneId is cached inside the TimeZone object itself:

```
public ZoneId toZoneId() {
        ZoneId zId = zoneId;
        if (zId == null) {
            zoneId = zId = toZoneId0();
        }
        return zId;
    }
```

Reply:
Good find, thank you!